plugins/woocommerce/tests/php/includes/class-wc-shortcodes-test.php

```diff
@@ -476,4 +476,30 @@ public function test_product_page_shortcode_protected_product() {
 
 		$this->assertStringContainsString( 'This content is password protected', $product_page );
 	}
+
+	/**
+	 * Ensure that when both [woocommerce_cart] and [woocommerce_checkout] shortcodes
+	 * are rendered together, no redirect to the cart page occurs.
+	 */
+	public function test_combined_cart_and_checkout_shortcodes_do_not_redirect() {
```

Comment:
It looks like there was some linting issues were reported in the function that were refactoring anyway, so I'm not sure whether you have it setup correctly in your IDE but might be worth doing it to avoid further linting failures.